<launch>  
	<!-- Camera Setup -->
	<!--<param name="CALIB_PATH" value="$(arg calibpath)"/>-->
	
	<group ns="stereo" >
		<!-- Open ALL FIREWIRE cameras with the same configuration -->
		<node pkg="multi_camera1394" type="multi_camera1394_node" name="multi_camera1394_node" >
			<remap from="camera_0" to="left" />
			<remap from="camera_1" to="right" />
			<param name="guid_0" value="00b09d0100acc96d" />
			<param name="guid_1" value="00b09d0100acc969" />
			<param name="camera_type" value="1"/>
			<param name="video_mode" value="1280x960_mono8"/>
			<param name="frame_rate" value="12.5"/>    
			<param name="enableEmbeddedTimestamp" value="true"/>
			<param name="auto_shutter_maxvalue" value="500"/>
			<!--<param name="auto_gain_maxvalue" value="30"/>-->
			<!--<param name="camera_info_url" value="$(arg 1394calibpath)"/>-->
			<!--<rosparam param="multi_frame_id">[left, right]</rosparam>-->
			<!--<rosparam param="multi_guid">[00b09d0100acc96d, 00b09d0100acc969]</rosparam>-->
			<!--<rosparam command="load" file="cameras.yaml" />-->
		</node>
	</group>

	<!-- Manual Camera Tuning -->
	<!--<node pkg="dynamic_reconfigure" name="reconfigure_gui" type="reconfigure_gui"/>-->


	<!-- Publish Undistorted Images -->
	<!-- (note it is needed to set the namespace on camera with ns="") -->
	<!--<node ns="stereo" pkg="stereo_image_proc" type="stereo_image_proc" name="proc" />-->
 

	<!-- Display Camera -->
	<!-- image_view expects images on /image need to remap to /camera/image_* -->
	<!--<node pkg="image_view" type="image_view" name="left_image_view" >
 		<remap from="image" to="/stereo/left/image_raw"/>
	</node>
	
	<node pkg="image_view" type="image_view" name="right_image_view" >
 		<remap from="image" to="/stereo/right/image_raw"/>
	</node>-->

</launch>
